src/templates/app/new_account/virtual.jsx

```diff
@@ -24,17 +24,15 @@ const Virtual = () => (
                         {it.L('Strong passwords contain at least 8 characters, combine uppercase and lowercase letters, numbers, and symbols.')}
                     </p>
                 </div>
-
+            </Fieldset>
+            <div id='consent_checkbox'>
                 <FormRow
                     type='checkbox'
-                    checked
                     id='email_consent'
-                    row_class='invisible'
                     label_row_id='email_consent_label'
-                    label={it.L('Receive news and special offers')}
+                    label={it.L('I want to receive updates on binary.com products, services, and events.')}
```

Comment:
binary.com should be a placeholder with it.website_name

Reply:
binary.com now replaced with the placeholder.